docs/sources/flow/reference/stdlib/join.md

```diff
@@ -0,0 +1,24 @@
+---
+aliases:
+- ../../configuration-language/standard-library/join/
+title: join
+---
+
+# join
+
+`join` produces a string by concatenating all of the elements of the specified list of strings with the specified separator.
```

Comment:
```suggestion
`join` produces a string by concatenating all of the elements of a list of strings with a separator.
```

Could we simply this a little?

Reply:
@clayton-cornell I was not sure if the suggested change from you was enough, I commit an alternative description for join. please check.

Reply:
Yeah that's much better. One small suggestion on the rework to use code for join.